spec/registry.md

```diff
@@ -619,6 +619,23 @@ All other values produce an _Invalid Expression_ error.
 > or the type `com.ibm.icu.util.CurrencyAmount` can be used to set the currency and related
 > options (such as the number of fraction digits).
 
+### Non-Negative Integer Options
+
+Some _options_ of number _functions_ are defined to take a "non-negative integer".
+Implementations of number _functions_ use these _options_ to control aspects of numeric display
+such as the number of fraction, integer, or significant digits.
+
+A "non-negative integer" is an _option_ value that the _function_ evaluates as an integer 
+greater than or equal to zero.
+Implementations MAY define an upper limit on the resolved value 
+of a non-negative integer option consistent with that implementation's practical limits.
+
+The implementation MAY accept any implementation-defined type as the value.
+Implementations MUST accept the value as a _literal_.
```

Comment:
I think this is slightly ambiguous, or at least it is to my non-native ears. I'd suggest reversing this sentence, so that it's clear that implementations must accept *at least* literals.

```suggestion
Implementations MUST accept a _literal_ as the value.
```